gatsby-node.js

```diff
@@ -99,21 +103,13 @@ exports.onCreateNode = ({ node, getNode, actions }) => {
     }
 
     localUrls.map(url => {
-      let newUrl = url.replace('.md', '/');
+      let newUrl = url.replace(/(\/index)?\.md/, '/');
       newUrl = `/${URL.resolve(nodePath, newUrl)}`;
       html = html.replace(url, newUrl);
       return true;
     });
 
     node.internal.content = html;
-    if ('index' === path.basename(nodePath)) {
```

Comment:
Could you keep that for now please? IIRC it's because we used to have such links, so it's to prevent a 404 for old links.

Reply:
It's not working. The code for managing these nodes has been removed. Maybe we can redo it in another PR?

Reply:
The code was this: https://github.com/api-platform/website/blob/f0a81f24a4bdfcf1422b4c3887463114371a61f1/gatsby-node.js#L129-L141